Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -4,7 +4,9 @@ import MapboxMaps
 @objc(Custom2DPuckExample)
 public class Custom2DPuckExample: UIViewController, ExampleProtocol {
 
+    internal var toggleAccuracyRadiusButton: UIButton?
```

Comment:
It's fairly common for properties like this to be implicitly unwrapped: `UIButton!`

however, you could also just initialize the button directly and make this a let: `let toggleAccuracyRadiusButton = UIButton(frame: CGRect.zero)`